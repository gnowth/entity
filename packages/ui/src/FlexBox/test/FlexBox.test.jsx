import 'jest-styled-components';
import renderer from 'react-test-renderer';
import React from 'react';

import UIFlexBox from '..';

it('renders correctly', () => {
  const tree = renderer.create(
    <UIFlexBox />,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
