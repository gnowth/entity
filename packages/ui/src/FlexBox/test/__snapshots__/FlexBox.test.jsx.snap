// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="c0"
/>
`;
